eventdata/runners/kibana_runner.py

```diff
@@ -51,6 +51,7 @@ async def kibana(es, params):
         "meta_data" - Dictionary containing meta data information to be carried through into metrics.
```

Comment:
if you don't go for my alternative above, let's document the expectation for `params`